ikasaneip/persistence/src/main/java/org/ikasan/persistence/hibernate/IkasanSybaseASE157Dialect.java

```diff
@@ -0,0 +1,23 @@
+package org.ikasan.persistence.hibernate;
+
+import org.hibernate.dialect.SybaseASE157Dialect;
+import org.hibernate.dialect.pagination.LimitHandler;
+import org.hibernate.engine.spi.RowSelection;
+
+/**
+ * Created by Ikasan Development Team on 27/12/2017.
+ */
+public class IkasanSybaseASE157Dialect extends SybaseASE157Dialect
```

Comment:
Please remove this class from Ikasan:
As discussed under:
https://github.com/ikasanEIP/ikasan/pull/410